src/navigator.js

```diff
@@ -271,6 +273,21 @@ $.extend( $.Navigator.prototype, $.EventSource.prototype, $.Viewer.prototype, /*
             }
         }
     },
+    /**
+      /* Flip navigator element
+      */
+    toggleFlip: function() {
+      this.viewport.flipped = !this.viewport.flipped;
```

Comment:
I imagine here you can call `this.viewport.toggleFlip()`.

Reply:
Also, with the addition Viewport.setFlip, this function probably needs to become Navigator.setFlip`.

Reply:
you mean, just rename the function?

    setFlip: function() {
      this.viewport.toggleFlip();

      this.setDisplayTransform(this.viewer.viewport.flipped ? "scale(-1,1)" : "scale(1,1)");
      this.viewport.viewer.forceRedraw();
    },